src/components/vm/snapshots/vmSnapshotsCreateModal.jsx

```diff
@@ -124,14 +177,15 @@ export class CreateSnapshotModal extends React.Component {
 
     render() {
         const Dialogs = this.context;
-        const { idPrefix } = this.props;
-        const { name, description, submitted } = this.state;
+        const { idPrefix, isExternal, vm } = this.props;
+        const { name, description, memoryPath, submitted } = this.state;
         const validationError = this.onValidate(submitted);
 
         const body = (
             <Form onSubmit={e => e.preventDefault()} isHorizontal>
                 <NameRow name={name} validationError={validationError} onValueChanged={this.onValueChanged} />
                 <DescriptionRow description={description} onValueChanged={this.onValueChanged} />
+                {isExternal && vm.state === 'running' && <MemoryPathRow memoryPath={memoryPath} onValueChanged={this.onValueChanged} />}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1280-20240125-153407-61755b5b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)